RELEASE-NOTES.md

```diff
@@ -1,38 +1,40 @@
-  * Removed `snow snowpark function create` and `snow snowpark function update`. Procedures can be deployed using `snow snowpark deploy`.
+  * Removed `snow snowpark function create` and `snow snowpark function update`. Functions can be deployed using `snow snowpark deploy`.
   * Removed `snow snowpark procedure create` and `snow snowpark procedure update`. Procedures can be deployed using `snow snowpark deploy`.
-  * Procedures and functions use single zip artifact for all functions and procedures.
+  * Procedures and functions use single zip artifact for all functions and procedures in project.
   * Changed path to coverage reports on stage, previously created procedures with coverage will not work, have to be recreated.
   * Previously created procedures or functions won't work with `deploy` command due to change in stage path of artefact. Previous code will remain under old path on stage.
+  * Coverage commands are now under `snow snowpark coverage`.
+  * Package commands are now under `snow snowpark package`.
+
 * Snowpark Containers services commands
   * `compute-pool` commands and its alias `cp` were renamed to `pool` commands.
   * `jobs` commands were renamed to `job`.
   * `services` commands were renamed to `service`
   * `pool`, `job` and `service` commands were moved from `snowpark` group to a new `containers` group.
-* `snow snowpark registry` was replaced with `snow registry` command.
-* `snow connection test` now outputs all connection details (except for the password), along with connection status
-* From `snow streamlit deploy` moved following parameters to `snowflake.yml`:
-  * Argument:
-    * `streamlit-name`
-  * Flags:
-    * `--file`
-    * `--stage`
-    * `--env-file`
-    * `--pages-dir`
-* `init` commands for functions and procedures create new project in new directory instead of using current working directory.
-* Moved `snow stage` from top-level to `snow object` subgroup
-* `snow warehouse status` is now `snow object show warehouse`
-* Introduced `snow object` group with `show`, `describe` and `drop` commands for: compute pools,
-databases, tables, warehouses, functions, procedures, roles, schemas, services, jobs and streamlits
+  * `snow snowpark registry` was replaced with `snow registry` command.
+  * `snow connection test` now outputs all connection details (except for the password), along with connection status
+
+* Streamlit changes
+  * `snow streamlit deploy` is requiring `snowflake.yml` project file with a Streamlit definition.
+  * `snow streamlit describe` is now `snow object describe streamlit`
+  * `snow streamlit list` is now `snow object show streamlit`
+  * `snow streamlit drop` is now `snow object drop streamlit`
 
 
 ## New additions
 * `--temporary-connection` flag, that allows you to connect, without anything declared in config file
 * Added project definition for Streamlit
+* Added `snow streamlit get-url [NAME]` command that returns url to a Streamlit app.
```

Comment:
Added this because `streamlit describe` also returned app url. Now this functionality is separated